lib/covid19_questionnaire_web/schemas/questionnaire_request.ex

```diff
@@ -45,15 +45,13 @@ defmodule Covid19QuestionnaireWeb.Schemas.QuestionnaireRequest do
           "liver_disease" => true,
           "diabetes" => true,
           "immunosuppressant_disease" => 1,
-          "immunosuppressant_drug" => 1,
-          "cancer" => true,
```

Comment:
Same comment for "cancer": the frontend still send this variable.

Reply:
Corrected, thanks !